testing/py3-freezegun/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Contributor: Robert Sacks <robert@sacks.email>
+# Maintainer: Robert Sacks <robert@sacks.email>
+pkgname="py3-freezegun"
+_pyname="freezegun"
+pkgver="0.3.12"
+pkgrel=0
+pkgdesc="Let your Python tests travel through time"
+url="https://pypi.org/project/freezegun/"
+arch="noarch"
+license="Apache-2.0"
+depends="python3 py3-dateutil"
+makedepends="py3-setuptools py3-coverage py3-mock py3-pytest"
```

Comment:
It seems like `py3-coverage py3-mock py3-pytest` should be in `checkdepends`

Reply:
Fixed